api/message-utils.js

```diff
@@ -1,9 +1,9 @@
 const _ = require('underscore'),
-      db = require('./db-nano'),
+      db = require('./db-pouch'),
       taskUtils = require('task-utils');
 
 const getTaskMessages = function(options, callback) {
-  db.medic.view('medic', 'tasks_messages', options, callback);
+  db.medic.query('medic/tasks_messages', options, callback);
```

Comment:
If you felt like converting to promises at the same time that would be awesome, but ok if you don't.

Reply:
Definitely don't.  If we want to convert to promises that's fine, but in this PR it would completely obscure the meaningful change being made.